logic.py

```
@@ -672,7 +709,66 @@ def _is_candidate_valid(col):
+                        <thead style='background: #f8f9fa;'>
+                            <tr>
+                                <th style='padding:4px; color: #444; background: #f1f3f4;'>Variable</th>
+                                <th style='padding:4px; color: #444; background: #f1f3f4;'>VIF</th>
+                            </tr>
+                        </thead>
+                        <tbody>{vif_body}</tbody>
+                    </table>
+                    <small style='color: #666;'>Threshold: >{vif_threshold} indicates high collinearity.</small>
+                </div>
+                """
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)